src/components/DatePicker/DatePicker.tsx

```diff
@@ -100,8 +98,8 @@ export const MetDatePicker: FC<MetDatePickerProps> = ({
         {defaultCalendarIcon}
         <span className={dateFontClass}>
           {type === Type.FULL
-            ? getFullFormat(selectedDate)
-            : getShortFormat(selectedDate)}
+            ? moment(selectedDate).format("DD.MM.YYYY")
+            : moment(selectedDate).format("MMMM YYYY")}
```

Comment:
Format setting better take out in constants.
And take possibility to user pass it in props.